src/main/external-resources/renderers/Google-ChromecastUltra.conf

```diff
@@ -41,12 +41,5 @@ Supported = f:m4a            n:2      a:aac-lc|he-aac|mp3   m:audio/mp4
 Supported = f:wav            n:2      a:lpcm                m:audio/x-wav
 Supported = f:weba           n:2      a:opus|vorbis         m:audio/webm
 
-# Supported image formats:
```

Comment:
OK, thanks.
Most of them will work perfectly with your PR, and very few, like some Chromecast ones, will work not optimally, but anyway work, so that just a little fine tuning that perhaps doesn't deserve the effort ?